modules/apps/dynamic-data-mapping/dynamic-data-mapping-uad-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/uad/display/test/DDMFormInstanceRecordUADDisplayTest.java

```diff
@@ -33,13 +34,24 @@
  */
 @RunWith(Arquillian.class)
 public class DDMFormInstanceRecordUADDisplayTest
-	extends BaseUADDisplayTestCase<DDMFormInstanceRecord> {
+	extends BaseUADDisplayTestCase<DDMFormInstanceRecord>
+	implements WhenHasStatusByUserIdField<DDMFormInstanceRecord> {
 
 	@ClassRule
 	@Rule
 	public static final AggregateTestRule aggregateTestRule =
 		new LiferayIntegrationTestRule();
 
+	@Override
+	public DDMFormInstanceRecord addBaseModelWithStatusByUserId(
+			long userId, long statusByUserId)
+		throws Exception {
+
+		return DDMFormInstanceRecordTestUtil.
+			addDDMFormInstanceRecordWithStatusByUserIdAndNoValues(
+				_group, userId, statusByUserId);
```

Comment:
@algobob the `addDDMFormInstanceRecordWithStatusByUserIdAndNoValues` method's signature is `Group group, long statusByUserId, long userId`, as you can see [here](https://github.com/carolmariaabb/liferay-portal/blob/2214e4506340a985164daafa585ca5a14cdabbfe/modules/apps/dynamic-data-mapping/dynamic-data-mapping-test-util/src/main/java/com/liferay/dynamic/data/mapping/test/util/DDMFormInstanceRecordTestUtil.java#L72-L82). 

Therefore, to ensure correct testing, the call should be:

```java
return DDMFormInstanceRecordTestUtil.
	addDDMFormInstanceRecordWithStatusByUserIdAndNoValues(
		_group, statusByUserId, userId);
```